src/components/NewFilm/NewFilm.js

```diff
@@ -72,7 +74,7 @@ export class NewFilm extends Component {
       });
     } else {
       this.setState(initialState);
-      onAdd(formValue);
+      onAdd({ ...formValue, id: formValue.title });
```

Comment:
generate unique id instead of title

Tip: you can generate id in reducer

Reply:
> generate unique id instead of title
> 
> Tip: you can generate id in reducer

refactored